### DISCUSSION TITLE
Replication DB 필요

### DISCUSSION THREAD

**Discussion by laigasus** [cloud2scape/.github#28] [on-일반]
로컬에서 RW. RO 두 DB가 연결된 구성환경이 필요해요오
환경 세팅 도와주실 @tensorplumber 구해요~~

가능하신지요

**Comment by gimsoomin** [comment]
낮이나 밤이나 열심히 하겠습니다

**Comment by laigasus** [comment]
이슈 할당했습니다.
업무 참고 드려요~
https://github.com/cloud2scape/on-premise/issues/1
